DocumentsFromSnapshotMigration/src/main/java/com/rfs/RfsMigrateDocuments.java

```diff
@@ -118,14 +118,27 @@ public static class Args {
 
         @Parameter(required = false,
         names = "--documents-per-bulk-request",
-        description = "Optional.  The number of documents to be included within each bulk request sent, default 1000")
-        int numDocsPerBulkRequest = 1000;
+        description = "Optional.  The number of documents to be included within each bulk request sent. Default no max (controlled by documents size)")
+        int numDocsPerBulkRequest = Integer.MAX_VALUE;
+
+        @Parameter(required = false,
+            names = "--documents-size-per-bulk-request",
+            description = "Optional. The maximum aggregate document size to be used in bulk requests in bytes. " +
+                "Note does not apply to single document requests. Default 1 MiB")
```

Comment:
When do we do single document requests?

Reply:
This is still a bulk request, but when a single document size is > documents-size-per-bulk-request, we still send it as a single document bulk request